layouts/conference.html

```diff
@@ -36,7 +36,7 @@
                 <div class="solo">
                     <h2>Speakers</h2>
 
-                    <p>View Source features a world-class group of speakers whose talks will expose best practices and practical, innovative development techniques. In addition, we will have a daily set of group discussions with experts on a variety of hot topics.</p>
+                    <p>View Source features 16 world-class speakers (plus discussions and Q&A breakouts), exposing best practices and new technologies. Check out our confirmed speakers below, and <a href="{{ conference_data.folder }}/newsletter" title="Get email updates">stay tuned</a> for upcoming additions to our stellar lineup.</p>
```

Comment:
This is so nit-picky as to be ridiculous... "lineup" ends up by itself on a new line. If you edit the text to be `stellar&nbsp;lineup.` it will drag "stellar" down with it for a little more polish.

Not necessary to fix this to have this PR merged.